src/main/resources/application.yml

```diff
@@ -74,4 +74,4 @@ management:
       cache:
         time-to-live: 2000ms
 
-graceful.shutdown.enabled: true
+server.shutdown.grace-period: 10s
```

Comment:
I'm guessing you copied this from my PR? Sorry it's wrong! https://docs.spring.io/spring-boot/docs/2.3.0.RELEASE/reference/htmlsingle/#boot-features-graceful-shutdown